src/app/authentication/register/register.component.html

```diff
@@ -1,36 +1,57 @@
-<div class="container">
-    <div class="row d-flex justify-content-center">
-        <div class="col-6">
-            <form [formGroup]="registerForm" (ngSubmit)="onSubmit()">
-                <div class="form-group">
-                    <label for="username">Username</label>
-                    <input class="form-control" type="text" formControlName="username" placeholder="Enter your username" name="username">
-                </div>
-
-                <div class="form-group">
-                    <label for="password">Password</label>
-                    <input class="form-control" type="password" formControlName="password" placeholder="Enter your password" name="password">
-                </div>
-
-                <div class="form-group">
-                    <label for="confirmPassword">Confirm password</label>
-                    <input class="form-control" type="password" formControlName="confirmPassword" placeholder="Confirm your password" name="confirmPassword">
-                </div>
-
-                <div class="form-group">
-                    <label for="email">Email</label>
-                    <input class="form-control" type="email" formControlName="username" placeholder="Enter your email">
-                </div>
-
-                <div class="form-group">
-                    <!-- Dodati disabled ako forma nije ispravna -->
-                    <!-- Ako forma nije ispravna, da pocrvene polja koja nisu ispravna -->
-                    <!-- registerForm.controls['polje_neko'].valid, .touched -->
-                    <button class="btn btn-primary" type="submit">
-                        Submit
-                    </button>
-                </div>
-            </form>
+<div class="container sign-in-page">
```

Comment:
sign-in ili sign-up? Jer je registracija, pa mislim da je sign-up.
Mozda i da promenis klasu, da nema u imenu page, nego ili samo sign-up da bude ili mozda nesto sto ce blize odrediti.
Sign-up-card ili tako nesto, sto ce lako da te asocira.
Jer page je ipak ta sama komponenta, i onda nije bas dobro da se tako nazivaju klase

Reply:
Moze, promenicu samo u sign-up